datafusion/sql/src/statement.rs

```diff
@@ -230,8 +239,149 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 with_options,
                 if_not_exists,
                 or_replace,
-                ..
```

Comment:
I removed the `..` from this match statement and then let github copilot write the checks for all the clauses.